src/lib-mail/mail-html2text.c

```diff
@@ -87,7 +87,7 @@ parse_tag_name(struct mail_html2text *ht,
 		return 8 + 1;
 	}
 
-	if (size >= 10 && i_memcasecmp(data, "blockquote", 10) == 0 &&
+	if (size >= 11 && i_memcasecmp(data, "blockquote", 10) == 0 &&
```

Comment:
I wonder if this code is making an assumption about nul-termination.

Reply:
if you look at similar checks above, it will become clear that this is a simple slip of the tongue.

Reply:
The only caller of this function is parse_data(), which passes in the size `size - i - 1`.  Since i >= 0, the size passed to parse_tag_name() will always be < the actual allocated buffer size.

With that said, keeping that sort of assumption would be a bad idea. Based on the code higher up for script and style, I think your patch is good. I'll post it for internal review & CI.

Thanks.

Reply:
Good afternoon.
I hope you are doing well.
How are you doing on this PL?

The project has no approved PR for a long time.
Perhaps I missed some sad news, or this is the normal state of affairs for the project.

Reply:
Sorry these take very long for us. I'll see what we can do.

Reply:
Also, we merge everything in our internal gitlab and close the pull requests. So there are no merged pull requests in github.

Reply:
Thank you for your prompt response.